python/sauron-core/core/analysis/classifier.py

```
@@ -0,0 +1,24 @@
+from typing import Dict
+
+class AIImageClassifier:
+    def __init__(self):
+        self.classifier = SVC(probability=True)
+        self.scaler = StandardScaler()
+        self.feature_columns = [
+            'entropy', 'correlation_h', 'correlation_v',
+            'noise', 'wavelet_total', 'contrast'
+        ]
+    
+    def prepare_features(self, metrics: Dict[str, float]) -> np.ndarray:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)